internal/driver/glfw/loop.go

```diff
@@ -130,6 +130,15 @@ func (d *gLDriver) runGL() {
 			if f := l.OnStopped(); f != nil {
 				l.QueueEvent(f)
 			}
+
+			for len(funcQueue.Out()) > 0 {
```

Comment:
A comment here could be nice too to explain the goal of this code block (I assume it's so that goroutines blocked on a `fyne.DoAndWait` will not be blocked forever)